doc/depthFirstSearch/pgr_depthFirstSearch.rst

```diff
@@ -10,9 +10,8 @@
 pgr_depthFirstSearch - Experimental
 ===============================================================================
 
-``pgr_depthFirstSearch`` — Returns the traversal order(s) using Depth First
-Search algorithm. In particular, the Depth First Search algorithm and the
-Undirected DFS algorithm implemented by Boost.Graph.
+``pgr_depthFirstSearch`` — The Depth First Search and the Undirected DFS algorithm
+implemented by Boost.Graph.
```

Comment:
Returns a depth first search traversal of the graph. The graph can be directed or undirected.